_layouts/default.html

```diff
@@ -85,78 +83,16 @@ <h2><span>{{ page.title }}</span></h2>
-                                                    <a href="#"><img src="../assets/images/works/img1.jpg" alt="" class="img-fluid"></a>
-                                                </div>
-                                                <div class="wb"><a href="#">Beautiful Day With Friends..</a> <span class="post-date">Jun 15, 2017</span></div>
-                                            </li>
-                                            <li class="clearfix">
-                                                <div class="wi">
-                                                    <a href="#"><img src="../assets/images/works/img2.jpg" alt="" class="img-fluid"></a>
-                                                </div>
-                                                <div class="wb"><a href="#">Nature valley with cooling..</a> <span class="post-date">Jun 10, 2017</span></div>
-                                            </li>
-                                            <li class="clearfix">
-                                                <div class="wi">
-                                                    <a href="#"><img src="../assets/images/works/img3.jpg" alt="" class="img-fluid"></a>
-                                                </div>
-                                                <div class="wb"><a href="#">15 Best Healthy and Easy Salad..</a> <span class="post-date">Jun 8, 2017</span></div>
-                                            </li>
-                                        </ul>
-                                    </aside>
-
-                                    <!-- Text widget-->
-                                    <aside class="widget">
-                                        <div class="widget-title">Text Widget</div>
-
-                                        <p class="text-muted text-widget-des">Exercitation photo booth stumptown tote bag Banksy, elit small batch freegan sed. Craft beer elit seitan exercitation, photo booth et 8-bit kale chips proident chillwave deep v laborum. Aliquip veniam delectus, Marfa eiusmod Pinterest in do umami readymade swag. </p>
-
+                                            <ul>
+                                                {% assign a = site.posts | group_by:"categories" | sort:"size" %}
+                                                {% for category in a %}
+                                                    {% assign category_slug = category['name'] | slugify: "raw" | remove: '["' | remove: '"]' | capitalize %}
+                                                    <li class="tagcloud"><a href={{ "../" | append: category_slug | append: "/" }}>{{ category_slug }} ({{ category['size'] }})</a></li>
+                                                {% endfor %}
+                                            </ul>
                                     </aside>
 
                                     <!-- Archives widget-->
```

Comment:
Lets remove this as well, it's not doing anything right now, just static blank links.

Reply:
Looking at the UI now, I think I like the widget categories instead of the drop down on the left navigation, less, buttons to click and its just there and easy to navigate. We can discuss it more too.